### PR TITLE
feat(console): cloud preview welcome page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

This PR initializes the foundation of the cloud preview pages and adds the welcome page.

## Testing

Feb-09-2023 17-33-56

Updated:
image

### CODE REVIEW

**Comment by xiaoyijun** [comment]
Conver to draft for the component refactoring content can be separated into small prs.

**Comment by xiaoyijun** [comment]
Convert to draft since the tech design is under review.

File: packages/console/src/App.tsx [hunk 1]
```diff
@@ -73,7 +75,10 @@ const Main = () => {
             <Route path="callback" element={<Callback />} />
             <Route path="welcome" element={<Welcome />} />
             <Route element={<AppLayout />}>
-              <Route path="/cloud-preview" element={<CloudPreview />} />
+              <Route path="cloud-preview" element={<CloudPreview />}>
```
Comment (wangsijie) [inline packages/console/src/App.tsx:78]: questionnaire?

**Review by wangsijie — APPROVED** [review]
Overall LGTM

File: packages/console/src/App.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/App.tsx:78]: The questionnaire is part of the preview content and is under the cloud preview page.

File: packages/console/src/pages/CloudPreview/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,16 @@
+.cloudPreview {
+  flex-grow: 1;
+  overflow: hidden;
+}
+
+.page {
```
[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/pages/CloudPreview/index.module.scss:6]: It's used in the page content as a page layout.

[the remark this answers is not in the document]
Comment (xiaoyijun) [reply packages/console/src/pages/CloudPreview/index.module.scss]: updated.

**Pull request merged**